src/frontends/pytorch/src/op_table.cpp

```diff
@@ -247,12 +248,18 @@ OP_CONVERTER(translate_constant_pad_nd_fx);
 OP_CONVERTER(translate_chunk_fx);
 OP_CONVERTER(translate_div_fx);
 OP_CONVERTER(translate_expand_fx);
+OP_CONVERTER(translate_fake_quantize_per_channel_affine_fx);
+OP_CONVERTER(translate_fake_quantize_per_tensor_affine_fx);
 OP_CONVERTER(translate_full_fx);
 OP_CONVERTER(translate_gelu_fx);
 OP_CONVERTER(translate_group_norm_fx);
 OP_CONVERTER(translate_index_fx);
 OP_CONVERTER(translate_layer_norm_fx);
 OP_CONVERTER(translate_leaky_relu_fx);
+OP_CONVERTER(translate_log_sigmoid);
+OP_CONVERTER(translate_log_softmax_fx);
+OP_CONVERTER(translate_max_dim);
```

Comment:
You never use this, why add here?

Reply:
Moved max_dim to TS and max_dim_fx to FX translation